### ISSUE TITLE
Clock face scrambled

### ISSUE THREAD

**Issue by airwin13** [geoffhuntoon/simple-customizable-clock#56]
The clockface is displaying the date, time & heart rate clearly but other features have disappeared or not displaying properly
![842B29D0-CD6A-4F2C-8B4B-93BA416D272C](https://user-images.githubusercontent.com/77254508/104141583-a2f0de00-540b-11eb-9b91-c831fee23ea5.jpeg)

**Comment by geoffhuntsgood** [comment]
Yes! My apologies, this was an issue that came up from the latest update for a few users. Please do the following to see if it restores the original functionality:

1. Go to the settings page and toggle the new switch labeled "Reset Settings."
2. If #1 doesn't work, please completely uninstall the watch face, remove it from your saved watch faces, and then reinstall it.

I apologize for the inconvenience, please let me know if either of these helps!
